a11y/README.md

```diff
@@ -0,0 +1,28 @@
+# SYZYGY PL - Accessibility Styleguide
+
+## HTML
+
+* Always declare language attribute -- globally on your `html` wrapper and locally on elements containing text in other languages.
+* Use HTML5 sectioning elements such as `header`, `main`, `footer` `aside`, `section` to create a comprehensive landmark structure of the website or application. Also, remember about headings hierarchy (`h1` - `h6`).
```

Comment:
We could also mension about microformates (https://schema.org/) for more detailed semantics. But I'm not sure if it's more about a11y or "nicer search results".

Reply:
Doczytam, bo nic o tym nie wiem :)